submissions/israelOladosu/database/transportation.sql

```diff
@@ -0,0 +1,84 @@
+CREATE DATABASE transporation;
+
+use transporation;
+
+-- classes table
+CREATE TABLE classes (
+    id INT NOT NULL AUTO_INCREMENT,
+    class_level INT NOT NULL,
+    PRIMARY KEY(id)
+);
+
+-- passangers table
+CREATE TABLE passengers (
+    id INT NOT NULL AUTO_INCREMENT,
+    firstname VARCHAR(50) NOT NULL,
+    lastname VARCHAR(50) NOT NULL,
+    sex VARCHAR(6) NOT NULL,
+    age INT,,
```

Comment:
Double `,`

Reply:
Sorry, I made the corrections in-between office work, not a good excuse though. 
But this time, I made sure I ran all the queries before pushing.

Thanks.